bayesian_benchmarks/data.py

```diff
@@ -753,6 +755,173 @@ class C(Classification):
         name, N, D, K = name, N, D, K
 
 
+path_to_mujoco_dataset = '/home/felix/Desktop/policy_destination'
```

Comment:
I would set this to `None` and throw a warning saying a user should specify the path to the dataset.

Reply:
Can you add a comment here saying that this is a temporary solution, and that we are working on making the data accessible from the internet.

Reply:
done